### PR TITLE
add xontrib entry for click-tabcompleter

### PR DESCRIPTION
**Pull request by Granitosaurus**
Hi,

I've made a generic tabcompleter for applications based on [click](http://click.pocoo.org/5/) command line interface framework, which seems to be very popular for newer python applications.

Pypi: https://pypi.python.org/pypi?name=xonsh-click-tabcomplete
Github: https://github.com/Granitosaurus/xonsh-click-tabcomplete

### CODE REVIEW

File: xonsh/xontribs.json [hunk 1]
```diff
@@ -71,6 +71,11 @@
   "url": "https://github.com/Granitas/xonsh-vox-tabcomplete",
   "description": ["Adds tabcomplete functionality to vox inside of xonsh."]
  },
+ {"name": "click_tabcomplete",
+  "package": "xonsh-click-tabcomplete",
+  "url": "https://github.com/Granitas/xonsh-click-tabcomplete",
```
Comment (mitnk) [inline xonsh/xontribs.json]: Did you just changed the usernmame of github? also the url of `xonsh-vox-tabcomplete`.

**Comment by laerus** [comment]
ty @Granitosaurus !

File: news/tabcomplete_for_click_apps.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+**Added:** 
+
+* Tab completion xontrib for python applications based on click framework.
```
Comment (astronouth7303) [inline news/tabcomplete_for_click_apps.rst]: Please rephrase this to make it more clear that this is a new xontrib and what that xontrib is called.

Comment (astronouth7303) [inline news/tabcomplete_for_click_apps.rst]: Nevermind, already merged.
